code/game/machinery/telecomms/machines/message_server.dm

```diff
@@ -24,7 +24,7 @@
 /obj/machinery/blackbox_recorder/attack_hand(mob/living/user)
 	. = ..()
 	if(stored)
-		user.put_in_hands(stored)
+		stored.forceMove(get_turf(src))
```

Comment:
Use `drop_location()` instead, here and everywhere else.

Reply:
i tried, i really did, i tried drop_location(stored) as well as stored.drop_location() .... all those did, was *delete* the items they were supposed to drop, like, completely vanish. force moving it however worked flawlessly

Reply:
I believe @Jared-Fogle meant `stored.forceMove(drop_location())`
If you're ever unsure of how to use a particular function or whatever, one way to quickly figure out is to search the codebase for example use cases! :)
![image](https://user-images.githubusercontent.com/29008542/103471426-3b24fe00-4d4e-11eb-9cdf-8bc6202c31f5.png)

If you search for the proc definition (or ctrl+click a function call if you're using vscode and the .dm extensions) you can sometimes see helpful comments explaining the function further.
![image](https://user-images.githubusercontent.com/29008542/103471462-c3a39e80-4d4e-11eb-910b-0d17b5ca6f3e.png)

Reply:
this is outdated, its already added in.